contract/src/lib.rs

```diff
@@ -404,6 +409,69 @@ impl BtcLightClient {
         }
     }
 
+    fn check_target(&self, block_header: &Header, prev_block_header: &ExtendedHeader) {
+        if prev_block_header.block_height % BLOCKS_PER_ADJUSTMENT != 0 {
```

Comment:
Isn't it should be `block_header.block_height`

Reply:
fixed: https://github.com/Near-One/btc-light-client-contract/pull/64/commits/47ed0af32e6b6bd8deafa20f2c90397a85dae1f9